src/components/timers/TimerList.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { DragDropContext, Droppable, Draggable } from 'react-beautiful-dnd';
+import './TimerList.css';
+
+const TimerList = ({ timers, onReorder, onRemove }) => {
+  const handleDragEnd = (result) => {
+    if (!result.destination) return;
+
+    const sourceIndex = result.source.index;
+    const destinationIndex = result.destination.index;
+
+    if (sourceIndex === destinationIndex) return;
+
+    onReorder(sourceIndex, destinationIndex);
+  };
+
+  
+  return (
+    <DragDropContext onDragEnd={handleDragEnd}>
+      <Droppable droppableId="timers">
```

Comment:
When I try to re-order, i get the following error: 
`Unable to find draggable with id: Stopwatch-0`

It does work if I remove StrictMode. Not sure exactly what the issue is, but it has to do with this I think: 
https://stackoverflow.com/questions/60029734/react-beautiful-dnd-i-get-unable-to-find-draggable-with-id-1